centml/sdk/config.py

```diff
@@ -4,19 +4,22 @@
 
 
 class Config(BaseSettings):
-
     # It is possible to override the default values by setting the environment variables
-    model_config = SettingsConfigDict(env_file=Path('.env'))
+    model_config = SettingsConfigDict(env_file=Path(".env"))
 
     CENTML_WEB_URL: str = os.getenv("CENTML_WEB_URL", default="https://app.centml.com/")
```

Comment:
this is not a comment on this PR. but I really don't understand why added these os.getenv() part. the whole point of using pydantic config is to void reading env vars like this.

Reply:
@johncalesp made some changes cuz he was blocked by env vars not being able to be read. He updated it with this PR: https://github.com/CentML/centml-python-client/pull/86

Reply:
Yeah it should, but John said it didn't work in the PR. We can look at this later if you think we should revert it.